### DISCUSSION TITLE
MultiQC

### DISCUSSION THREAD

**Discussion by gregdenay** [bio-raum/FooDMe2#18] [General]
@marchoeppner 

sooo I'm trying to add cutadapt to MultiQC, the module emits a report that mutliqc can parse out of the box.
I tried to add the report to the qc channel in the Illumina workflow:

```
multiqc_files  = multiqc_files.mix(CUTADAPT.out.report)
```

The JSON files are named with the `{sample}.cutadapt.json` pattern as recommended in the cutadatpt doc.

How do I get the report to show up in MultiQC? 
I checked the multiqc config in `,/conf` but it doesn't look like in the multiqc doc. Shouldn't there be a section per module?

**Comment by marchoeppner** [comment]
If MultiQC can see and interpret the report, it would include it, normally. You can check the work directory (the folder path is listed in the nextflow on-screen progress meter, in the square brackets on the left) to  see if the cutadapt reports really do show up in the folder. If yes, there is a chance something confuses MultiQC. 

A way to debug this would be to install Multiqc via Conda, take one of the reports (preferably just the report, nothing else, in a separate folder) and do:

```bash
multiqc .
```

If it does not find any reports, then something about the report is "wrong".

**Comment by gregdenay** [comment]
All the files (cutadapt + fastp) are in the work folder. When I run `multiqc .` from conda, then the report has a cutadapt section. 
When nextflow runs it, then no cutadapt. The only difference is the config file, isn't it?

**Reply by marchoeppner** [reply]
or the MultiQC version - the module uses 1.19; the latest version is 1.22. 

Can you update the container section with: 1.22.3--pyhdfd78af_0

and see if that helps?

**Reply by gregdenay** [reply]
It helped :)
